cardinal/read/read_archetype.go

```diff
@@ -7,8 +7,8 @@ import (
 
 	"github.com/argus-labs/starter-game-template/cardinal/game"
 	"pkg.world.dev/world-engine/cardinal/ecs"
+	"pkg.world.dev/world-engine/cardinal/ecs/entity"
 	"pkg.world.dev/world-engine/cardinal/ecs/filter"
-	"pkg.world.dev/world-engine/cardinal/ecs/storage"
 )
 
 type ArchetypeRequest struct {
```

Comment:
@technicallyty did we want to remove the Archetype stuff from `starter-game-template`, i recall us discussing that this a much more internal thing that Cardinal users don't need